### DISCUSSION TITLE
Time Line

### DISCUSSION THREAD

**Discussion by procraft7399** [artisan-roaster-scope/artisan#1677] [General]
After roasting when I go to stop artisan and save it, the timeline on the bottom moves completely off the screen, and you can see nothing but the first minute of the roast. Even after saving, if I load a new profile, it shows up the exact same way? Any help would be greatly appreciated
![97F4967D-520F-4DE4-A46E-EE4C2F7C1A92](https://github.com/user-attachments/assets/4852ffb5-d04e-425f-a145-82583a0b514b)
![B9E28515-68CE-4E11-93B0-382A121D014B](https://github.com/user-attachments/assets/7b6db003-6b73-4804-a126-514ef76da2b9)

**Comment by MAKOMO** [comment]
Did you check [Axis System](https://www.youtube.com/watch?v=QRAzYkmGYVw)

**Comment by procraft7399** [comment]
Thank you my Friend🙏 Fixed
